init/src/main.rs

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+//
+// Copyright (c) 2024 Intel Corporation.
+//
+// Author: Peter Fang <peter.fang@intel.com>
+
+#![no_std]
+#![no_main]
+
+extern crate alloc;
+use alloc::ffi::CString;
+use alloc::string::String;
+use buddy_system_allocator::*;
+use core::ffi::CStr;
+use core::panic::PanicInfo;
+use syscall::{exec, exit, opendir, readdir, DirEnt, SysCallError, F_TYPE_FILE};
+
+const HEAP_SIZE: usize = 64 * 1024;
+static mut HEAP: [u8; HEAP_SIZE] = [0; HEAP_SIZE];
+
+#[global_allocator]
+static HEAP_ALLOCATOR: LockedHeap<32> = LockedHeap::<32>::empty();
+
+#[no_mangle]
+pub extern "C" fn init_start() -> ! {
+    unsafe {
+        HEAP_ALLOCATOR
+            .lock()
+            .init(core::ptr::addr_of!(HEAP) as usize, HEAP_SIZE);
+    }
+
+    let bin = CString::new("/bin/").expect("Failed to create new string");
```

Comment:
```suggestion
    let bin = CString::new("/bin/").unwrap();
```

See https://github.com/coconut-svsm/svsm/pull/467#discussion_r1775205323